recipes-python/python-flake8-logging-format-native/files/0001-Remove-hard-dependency-on-nose.patch

```diff
@@ -0,0 +1,36 @@
+From f1bf907cc14f55edebac9d4633a4c1edec0f9f53 Mon Sep 17 00:00:00 2001
+From: S. Lockwood-Childs <sjl@vctlabs.com>
+Date: Fri, 4 Mar 2022 12:53:16 -0800
+Subject: [PATCH] Remove hard dependency on nose
+
+nose is not really a runtime dependency, but rather just a dependency
+for running the unit tests.
+
+Signed-off-by: S. Lockwood-Childs <sjl@vctlabs.com>
+---
```

Comment:
Could you please propose this change upstream first - I would rather like to remove the nose recipe as part of an update of this package

Reply:
And if upstream is dead, I'd like to see the corresponding `Upstream-Status` meta data entry here

Reply:
Ok I can try

Reply:
https://github.com/globality-corp/flake8-logging-format/pull/30

Reply:
Thx